var/spack/repos/builtin/packages/quantum-espresso/package.py

```diff
@@ -17,10 +17,11 @@ class QuantumEspresso(Package):
     """
 
     homepage = 'http://quantum-espresso.org'
-    url = 'https://gitlab.com/QEF/q-e/-/archive/qe-6.4.1/q-e-qe-6.4.1.tar.gz'
+    url = 'https://gitlab.com/QEF/q-e/-/archive/qe-6.5/q-e-qe-6.5.tar.gz'
     git = 'https://gitlab.com/QEF/q-e.git'
 
```

Comment:
Would you like to add yourself as a maintainer of the package? Just add `maintainers = ['naromero77']` and we'll ping you whenever someone opens a PR/issue related to `quantum-espresso`.